CMakeLists.txt

```diff
@@ -885,11 +885,20 @@ install(
     ${CMAKE_INSTALL_DATADIR}/mixxx
 )
 
-# Documentation
+# Licenses
 install(
   FILES
     ${CMAKE_CURRENT_SOURCE_DIR}/LICENSE
+    ${CMAKE_CURRENT_SOURCE_DIR}/COPYING
+  DESTINATION
+    ${CMAKE_INSTALL_DATADIR}/licenses/mixxx
+)
+
+# Documentation
+install(
+  FILES
     ${CMAKE_CURRENT_SOURCE_DIR}/README
+    ${CMAKE_CURRENT_SOURCE_DIR}/README.md
```

Comment:
Does it make sense to install both READMEs?

Reply:
I didn't think about this. Probably not, the README.md is only for GitHub.

Reply:
Removed.